docs/UserGuide.md

```diff
@@ -189,4 +189,11 @@ Action | Format, Examples
 **Edit** | `edit INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [t/TAG]…​`<br> e.g.,`edit 2 n/James Lee e/jameslee@example.com`
 **Find** | `find KEYWORD [MORE_KEYWORDS]`<br> e.g., `find James Jake`
 **List** | `list`
+**Group** | `group GROUP_NAME` <br> e.g., `group CS2103 Project Group`
```

Comment:
I think it might be better for the "Action" column to be a bit more descriptive about what the command does. Like for example, maybe "**Create Group**" instead of just "**Group**"? Otherwise, I am not sure if the summary would be very useful. This also occurs in other commands, not just this. Though this might be dependent on what the rest of the team expects from the summary. Maybe this can be brought up.

Reply:
I named again all actions, thanks :)